### ISSUE TITLE
Salzmann’s nodular corneal degeneration

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10871]
Preferred term label:
Salzman’s corneal degeneration

Synonyms
Salzmann’s degeneration, Salzmann’s nodular corneal degeneration, Salzmann’s nodular corneal dystrophy

Definition (free text, please give PubMed ID)
A corneal condition characterized by the formation of large, bluish, nodular lesions in the superficial corneal stroma, often associated with chronic keratitis or aging. These nodules may lead to visual impairment. [PMID:22228492](https://pubmed.ncbi.nlm.nih.gov/33212141/)

Parent term (use hpo.jax.org/app)
Corneal degeneration (HP:0007705)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Salzmann’s nodular corneal degeneration

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
I think the label should be as follows (because "Salzmann’s nodular corneal degeneration" is a disease entity rather than a phenotypic feature).


Adding new term: 
- ID: HP:6001230
- label: Salzmann corneal nodules
- parents: [Corneal degeneration (HP:0007705)](https://hpo.jax.org/app/browse/term/HP:0007705)
- pmid: PMID:22228492
- def: The formation of large, bluish, nodular lesions in the superficial corneal stroma, often associated with chronic keratitis or aging. These nodules may lead to visual impairment.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10871
- synonyms: Salzmann's nodules(NONE)|Salzmann nodular corneal dystrophy(NONE)|Salzmann nodular corneal degeneration(NONE)
